utils/sl/fake_scons/SCons/Script/__init__.py

```diff
@@ -231,7 +231,7 @@ def subst(self, val):
 
     def get(self, var, *_args, **_kw):
         """Fake get"""
-        return var
+        return [var]
```

Comment:
I believe this can be either a string or a list and code is expected to handle both.

You'll note there is a pylint regression that is likely a result of this change.

Reply:
Where do you see that ?

Reply:
Also. Can I fix this in a follow-up patch?  I think a better solution is just to add the libraries I need.  I don't think this particular variable can be a string.

Reply:
https://github.com/daos-stack/daos/pull/9381/files and the scroll all the way to the bottom.